gamemodes/attdef.pwn

```diff
@@ -450,45 +537,54 @@ public ServerOnPlayerDeath(playerid, killerid, reason)
 	}
 	else
 	{
-		switch(reason)
+		if(!RandomDeathMessages)
```

Comment:
I made it toggleable in /config and disabled by default. So there is no problem I suppose.